CMakeLists.txt

```diff
@@ -64,7 +64,9 @@ add_subdirectory(tests/cpp)
 
 install(FILES LICENSE DESTINATION docs/licensing COMPONENT licensing_genai RENAME LICENSE-GENAI)
 install(FILES third-party-programs.txt DESTINATION docs/licensing COMPONENT licensing_genai RENAME third-party-programs-genai.txt)
-set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+if(NOT DEFINED CPACK_ARCHIVE_COMPONENT_INSTALL)
+    set(CPACK_ARCHIVE_COMPONENT_INSTALL ON)
+endif()
```

Comment:
My bad. It works.

Will you extend readme?

Reply:
I suppose we need one more round of README re-work and maybe recommend this approach to be a default one to build GenAI.

Let's discuss it on tomorrow's meeting.